ztag/transforms/mongodb.py

```diff
@@ -0,0 +1,29 @@
+import copy
+from ztag.transform import ZGrab2Transform
+from ztag import protocols
+
+
+class MongoDBTransform(ZGrab2Transform):
+
+    name = "mongodb/banner"
+    port = None
+    protocol = protocols.MONGODB
+    subprotocol = protocols.MONGODB.BANNER
+
+    def __init__(self, *args, **kwargs):
+        super(MongoDBTransform, self).__init__(*args, **kwargs)
+
+    def _transform_object(self, obj):
+        zout = super(MongoDBTransform, self)._transform_object(obj)
+        results = self.get_scan_results(obj)
+        if not results:
+            return zout
+
+        zout.transformed = copy.deepcopy(results)
```

Comment:
What fields are actually in `results`? Line 24 suggests it has `error_message`, which I don't see in the schema at all.

Reply:
The fields in results are exactly what is in the schema. The `to_clean` code below was copied in with the rest of the transform boiler plate and I didn't notice that it should have been removed. Will remove lines 24-27.